dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -44,32 +47,32 @@ public class ObjectCreationNode extends Node {
      *
      * @param tree the NewClassTree
      * @param enclosingExpr the enclosing expression Node if it exists, or null
-     * @param constructor the constructor node
+     * @param identifier the identifier node
      * @param arguments the passed arguments
      * @param classbody the ClassDeclarationNode
      */
     public ObjectCreationNode(
```

Comment:
Comparing this to `NewClassTree` again, I notice that there are no type arguments.
Can you verify that [NewClassTree.getTypeArguments](https://github.com/eisop/jdk/blob/2f2d60b79b0035490403d785c2be7373b061f177/src/jdk.compiler/share/classes/com/sun/source/tree/NewClassTree.java#L62) returns the constructor type arguments, that is, if you have a constructor with constructor type parameters, `getTypeArguments` returns these, not the class type arguments?

Take this example:

````java
class Demo<T> {
    <S> Demo(Demo<S> other) {}
    
    static void test(Demo<String> in) {
        Object o = new <String>Demo<Number>(in);
    }
}
````

How do we access `Number` and how do we access `String` in that `ObjectCreationNode`?

Reply:
Thanks! I have tested this piece of code and found the behavior is correct, i.e., `NewClassTree.getTypeArguments` returns the construcotr type argument `String`.  On the other hand, if we want to access `Number`, we can use `getIdentifier()` of `NewClassTree` of `this.getIdentifier()`. And the name `identifier` looks consistent with `NewClassTree` and `ObjectCreationNode`, does this make sense to you?

Reply:
So in `ObjectCreationNode` there is no way to access the constructor type arguments.
It would seem worth it to at least document how to get to these, by going through the tree.